### PR TITLE
Develop done

### LINKED ISSUE #1: index.html has functions
## describe
to study about team project

## tasks
- [x] function1
- [x] function2

### PR DESCRIPTION
**Pull request by yonkookim**
made function1~2

resolve #1

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+let's start
+
+function1 done
```
Comment (yonggooo) [inline index.html]: add ! to the end of sentence

File: index.html [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+function2 done
```
Comment (yonggooo) [inline index.html]: add '!!!' to the end of sentence

**Review by yonggooo — CHANGES_REQUESTED** [review]
was not bad 
but need to edit just 2points.

File: index.html (hunk 1 above)
Comment (yonkookim) [reply index.html]: oops okay my boss!!!!

File: index.html (hunk 2 above)
Comment (yonkookim) [reply index.html]: 라져!!

**Review by yonggooo — APPROVED** [review]
good job!!!!

**Pull request merged**